aws/aws-sdk-js-v3
# Bugfix: function fromIni not working as intended within a Docker container

### Issue
Issue number, if available, prefixed with "#"

### Description
When using the `fromIni` function within a Docker container, the user's aws credentials can be sourced incorrectly. Example:
```javascript
this.client = new SNSClient({
      region: "us-west-2",
      credentials: fromIni({
        profile: "default",
        filepath: "~/.aws",
        configFilepath: process.env.AWS_CONFIG_FILEPATH
      })
});
```
Even though the file path is at "~/.aws", when ran in a docker container, the file pathing ends up being wrong.

ALSO: the fromIni function, if not passed a filepath, will assume the file path. The assumed file path, when in Docker, ends up being wrong.

### Testing
1. build a simple javascript app that uses AWS SNS
2. in the app, make sure you connect to AWS SNS using a similar method to the example above
3. start this app in Docker
4. you should see that you cannot connect to AWS SNS
5. now switch to this branch in your docker container and try using your AWS SNS app again
6. you should now see that AWS SNS is working and sending messages properly

### Additional context
Add any other context about the PR here.

---
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
